Kernel/Arch/aarch64/init.cpp

```diff
@@ -113,6 +115,9 @@ extern "C" [[noreturn]] void init()
     new (&bootstrap_processor()) Processor();
```

Comment:
In `init.cpp:106` should we update 'Serenity OS' to 'SerenityOS' for consistency? I can't add a comment at that exact line, so sneaking it in here.

Reply:
Seems unrelatedly to this PR. Feel free to make a PR for that (but that string is temporary anyways) 🙂

Reply:
Okay, sorry for the noise!